lib/ui/home/home_screen.dart

```diff
@@ -94,19 +61,20 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
                   avatar: profile?.avatarUrl ?? '',
                 ),
               ),
-              _buildPagerIndicator(mockSurveys.length),
+              _buildPagerIndicator(),
             ],
           );
         },
       );
 
-  Widget _buildPagerIndicator(int pagerIndicatorSize) => Consumer(
+  Widget _buildPagerIndicator() => Consumer(
         builder: (context, ref, child) {
           final visibleIndex = ref.watch(visibleIndexStream).value ?? 0;
+          final surveys = ref.watch(surveysStream).value ?? [];
```

Comment:
@hoangnguyen92dn we can get `surveys` from `_buildHomeContent()` method 😄

Reply:
@manh-t Indeed, but we still have to `watch` the `visibleIndex`, so that's why I added the `Consumer` and observed those streams in this method rather than passing surveys from the `_buildHomeContent()` 😄

Reply:
@hoangnguyen92dn I have no problem with `visibleIndex`, but `surveys` here is the same as the `surveys` from `_buildHomeContent()` so passing it via parameter is enough 🙆‍♂️

Reply:
@manh-t 🆗 I will make an update on the next PR, since we need to observe the `visibleIndex` to navigate to Survey Details on the `_buildHomeContent()` as well 🙏